<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeiboItem</class>
 <widget class="QWidget" name="WeiboItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeiboItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:60px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>149</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 繁黑體 Std B</family>
     <pointsize>14</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 14pt &quot;Adobe 繁黑體 Std B&quot;;
qproperty-alignment: 'AlignCenter';
background:transparent</string>
   </property>
   <property name="text">
    <string>邓超</string>
   </property>
  </widget>
  <widget class="QLabel" name="image1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 12pt &quot;Adobe 繁黑體 Std B&quot;;
font: 87 14pt &quot;Arial Black&quot;;background:transparent;
qproperty-alignment: 'AlignTop';</string>
   </property>
   <property name="text">
    <string>邓超</string>
   </property>
  </widget>
  <widget class="QLabel" name="date1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="text">
    <string>5月25日 16：50</string>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>700</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: 'AlignTop';
font: 10pt &quot;Adobe 黑体 Std R&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>总有一天会死在小花妹妹手里[泪][泪][泪] ​​​​</string>
   </property>
  </widget>
  <widget class="QLabel" name="image2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 12pt &quot;Adobe 繁黑體 Std B&quot;;
font: 87 14pt &quot;Arial Black&quot;;background:transparent</string>
   </property>
   <property name="text">
    <string>邓超</string>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>700</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: 'AlignTop';
font: 10pt &quot;Adobe 黑体 Std R&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>总有一天会死在小花妹妹手里[泪][泪][泪] ​​​​</string>
   </property>
  </widget>
  <widget class="QLabel" name="date2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="text">
    <string>5月25日 16：50</string>
   </property>
  </widget>
  <widget class="QLabel" name="image3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>620</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 12pt &quot;Adobe 繁黑體 Std B&quot;;
font: 87 14pt &quot;Arial Black&quot;;background:transparent;
qproperty-alignment: 'AlignTop';</string>
   </property>
   <property name="text">
    <string>邓超</string>
   </property>
  </widget>
  <widget class="QLabel" name="text3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>680</y>
     <width>700</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: 'AlignTop';
font: 10pt &quot;Adobe 黑体 Std R&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>总有一天会死在小花妹妹手里[泪][泪][泪] ​​​​</string>
   </property>
  </widget>
  <widget class="QLabel" name="date3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>660</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="text">
    <string>5月25日 16：50</string>
   </property>
  </widget>
  <widget class="QLabel" name="name4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>820</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 12pt &quot;Adobe 繁黑體 Std B&quot;;
font: 87 14pt &quot;Arial Black&quot;;background:transparent;
qproperty-alignment: 'AlignTop';</string>
   </property>
   <property name="text">
    <string>邓超</string>
   </property>
  </widget>
  <widget class="QLabel" name="text4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>880</y>
     <width>700</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: 'AlignTop';
font: 10pt &quot;Adobe 黑体 Std R&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>总有一天会死在小花妹妹手里[泪][泪][泪] ​​​​</string>
   </property>
  </widget>
  <widget class="QLabel" name="date4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>860</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="text">
    <string>5月25日 16：50</string>
   </property>
  </widget>
  <widget class="QLabel" name="image4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>820</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1020</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 12pt &quot;Adobe 繁黑體 Std B&quot;;
font: 87 14pt &quot;Arial Black&quot;;background:transparent;
qproperty-alignment: 'AlignTop';</string>
   </property>
   <property name="text">
    <string>邓超</string>
   </property>
  </widget>
  <widget class="QLabel" name="text5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1080</y>
     <width>700</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: 'AlignTop';
font: 10pt &quot;Adobe 黑体 Std R&quot;;
background:transparent</string>
   </property>
   <property name="text">
    <string>总有一天会死在小花妹妹手里[泪][泪][泪] ​​​​</string>
   </property>
  </widget>
  <widget class="QLabel" name="date5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1060</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="text">
    <string>5月25日 16：50</string>
   </property>
  </widget>
  <widget class="QLabel" name="image5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>1020</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>1200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="urlButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="text">
    <string>网页中查看</string>
   </property>
  </widget>
  <zorder>backgroundImage</zorder>
  <zorder>imageLabel</zorder>
  <zorder>name</zorder>
  <zorder>image1</zorder>
  <zorder>name1</zorder>
  <zorder>date1</zorder>
  <zorder>text1</zorder>
  <zorder>image2</zorder>
  <zorder>name2</zorder>
  <zorder>text2</zorder>
  <zorder>date2</zorder>
  <zorder>image3</zorder>
  <zorder>name3</zorder>
  <zorder>text3</zorder>
  <zorder>date3</zorder>
  <zorder>name4</zorder>
  <zorder>text4</zorder>
  <zorder>date4</zorder>
  <zorder>image4</zorder>
  <zorder>name5</zorder>
  <zorder>text5</zorder>
  <zorder>date5</zorder>
  <zorder>image5</zorder>
  <zorder>urlButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="webItem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>urlButton</sender>
   <signal>clicked()</signal>
   <receiver>WeiboItem</receiver>
   <slot>clickUrl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickUrl()</slot>
 </slots>
</ui>
